.github/workflows/package-windows.yml

```diff
@@ -0,0 +1,99 @@
+name: Package for Windows
+
+on:
+  push:
+    paths:
+    - '.github/workflows/package-windows.yml'
+    - 'installer/**'
+    - 'picard/**'
+    - 'po/**.po'
+    - 'resources/win10/**'
+    - 'scripts/package/*.ps1'
+    - 'scripts/pyinstaller/*'
+    - 'appxmanifest.xml.in'
+    - 'picard.ico'
+    - 'picard.spec'
+    - 'requirements*.txt'
+    - 'setup.py'
+    - 'tagger.py.in'
+    - 'win.version-info.txt.in'
+  create:
+    tags:
+    - 'release-*'
+
+jobs:
+  package:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
```

Comment:
Any reason we are using 3.7 for packaging?

Reply:
3.7 is the latest the works with current PyInstaller. They have mostly done the work, see https://github.com/pyinstaller/pyinstaller/issues/4311. So with the next PyInstaller release we should be able to try Python 3.7 for the macOS / Windows builds.